front/pages/w/[wId]/assistant/assistants.tsx

```diff
@@ -69,105 +49,47 @@ export const getServerSideProps = withGetServerSidePropsLogging<{
-    ? (context.query.view as PersonalAssitsantsView)
-    : "personal";
-
   return {
     props: {
       owner,
       subscription,
-      view,
       gaTrackingId: GA_TRACKING_ID,
     },
   };
 });
 
-export default function PersonalAssistants({
+export default function MyAssistants({
   owner,
   subscription,
-  view,
   gaTrackingId,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  const router = useRouter();
   const { agentConfigurations, mutateAgentConfigurations } =
     useAgentConfigurations({
       workspaceId: owner.sId,
-      agentsGetView: view === "personal" ? "list" : "workspace",
-      includes: view === "personal" ? ["authors"] : [],
+      agentsGetView: "list",
+      includes: ["authors"],
     });
 
   const [assistantSearch, setAssistantSearch] = useState<string>("");
   const [showDetails, setShowDetails] =
     useState<LightAgentConfigurationType | null>(null);
 
-  const viewAssistants = agentConfigurations.filter((a) => {
-    if (view === "personal") {
-      return a.scope === "private" || a.scope === "published";
-    }
-    if (view === "workspace") {
-      return a.scope === "workspace" || a.scope === "global";
-    }
-  });
-
-  const filtered = viewAssistants.filter((a) => {
+  const searchFilteredAssistants = agentConfigurations.filter((a) => {
     return subFilter(assistantSearch.toLowerCase(), a.name.toLowerCase());
   });
 
-  const [showRemovalModal, setShowRemovalModal] =
-    useState<LightAgentConfigurationType | null>(null);
-  const [showDeletionModal, setShowDeletionModal] =
-    useState<LightAgentConfigurationType | null>(null);
-
-  const tabs = [
-    {
-      label: "Personal & Shared Assistants",
-      href: `/w/${owner.sId}/assistant/assistants?view=personal`,
-      current: view === "personal",
-    },
-    {
-      label: "Company Assistants",
-      href: `/w/${owner.sId}/assistant/assistants?view=workspace`,
-      current: view === "workspace",
-    },
-  ];
-
-  const sendNotification = useContext(SendNotificationsContext);
-
-  const updateAgentUserList = async (
-    agentConfiguration: LightAgentConfigurationType,
-    listStatus: AgentUserListStatus
-  ) => {
-    const { errorMessage, success } = await updateAgentUserListStatus({
-      listStatus,
-      owner,
-      agentConfigurationId: agentConfiguration.sId,
-    });
-
-    if (success) {
-      sendNotification({
-        title: `Assistant ${
-          listStatus === "in-list" ? "added to" : "removed from"
-        } your list`,
-        type: "success",
-      });
-      await mutateAgentConfigurations();
-    } else {
-      sendNotification({
-        title: `Error ${
-          listStatus === "in-list" ? "adding" : "removing"
-        } Assistant`,
-        description: errorMessage,
-        type: "error",
-      });
-    }
-  };
-
   return (
     <AppLayout
       subscription={subscription}
       owner={owner}
+      hideSidebar
+      titleChildren={
+        <AppLayoutSimpleCloseTitle
```

Comment:
Why do we need to use `AppLayoutSimpleCloseTitle`? If I understand correctly, this is useful to have a "save"/"cancel" button in the top bar, which is not the case on this screen, right?

Reply:
Hmm as I understand it's just to have the close button, which we need there

Reply:
Indeed I missed that it's a "modal".